docs/blog/posts/2023/sr-linux-kubernetes-anycast-lab.md

```diff
@@ -0,0 +1,955 @@
+| **Lab**                   | [srl-labs/srl-k8s-anycast-lab][lab]                                                                                                                                                               |
+| **Version information**   | [`containerlab:0.42.0`](https://containerlab.dev/install/), [`srlinux:23.3.3`](https://github.com/nokia/srlinux-container-image),[`minikube v1.30.1`](https://minikube.sigs.k8s.io/docs/start/)|
+| **Authors**               | Míchel Redondo [:material-linkedin:][rd-linkedin]                                                                                                                                              |
+
+At the end of this blog post you can find a quick summary of the steps ([TL;DR](#tldr)).
+
+## Prerequisites
+
+The lab leverages [Containerlab](https://containerlab.dev/)  to spin up a Leaf/Spine Fabric coupled with [Minikube](https://minikube.sigs.k8s.io/) to deploy the Kubernetes cluster.
+
+[Docker engine](https://docs.docker.com/engine/install/) has to be installed on the host system.
+
+## Lab topology
+
+The goal of this lab is to provide users with an environment to test the network integration of a Kubernetes cluster with a Leaf/Spine SR Linux fabric.   
+
+
+<center markdown>![topology](sr-linux-kubernetes-anycast-lab/topology.svg){width=100%}</p>
+</small>Topology</small></center>
+
+
+The setup consists of:
+
+  - A Leaf/Spine Fabric: 2xSpines, 4xLeaf switches 
+  - Minikube kubernetes cluster (3 nodes) 
+  - Linux clients to simulate connections to k8s services (4 clients)
+
+Thanks to MetalLB, Kubernetes nodes establish BGP sessions with Leaf switches to announce the VIP of the service.
+
+Clients will simulate connections to the VIP by using curl. The service is a Nginx HTTP echo server, deployed as multiple pods in the Kubernetes Cluster.
+
+
+## Containerlab toplogy file
+
+The whole lab topology,  is declared in the Containerlab [`srl-k8s-lab.clab.yml`][clab-topo] file.
+
+Let's review the different components of our definition file:
+
+### Images
+First we look at the different container images that will be used:
+
+```yaml title="SR Linux and Client images"
+topology:
+  kinds:
+    srl:
+      image: ghcr.io/nokia/srlinux:23.3.3
+    linux:
+      image: ghcr.io/hellt/network-multitool
+  # -- snip --
+```
+
+We will use the latest [SR linux image](https://github.com/nokia/srlinux-container-image) as of today, that can be pulled as easily as `docker pull ghcr.io/nokia/srlinux:23.3.3`.
+
+[network-multitool](https://github.com/users/hellt/packages/container/package/network-multitool) versatile Linux image for the client.
+
+Kubernetes container images and container creation are directly managed by Minikube.
+
+### nodes
+And let's also review the definition of our Lab components: Leaf/Spine switches, K8s nodes and Linux clients:
+
+```yaml title="node definition"
+topology:
+  # -- snip --
+  nodes:
+    leaf1:
+      kind: srl
+      type: ixrd2l
```

Comment:
since 4 leaves are using ixrd2l, I'd encode this in a kind level, to specify d3ls only for spines and save some lines